src/main/java/us/dot/its/jpo/sec/helpers/HttpClientFactory.java

```diff
@@ -0,0 +1,17 @@
+package us.dot.its.jpo.sec.helpers;
```

Comment:
Looks like we abstracted some of this functionality into helper classes, but only have unit tests covering one of the new classes. Do we need to add additional tests?

Reply:
That seems reasonable. Unit tests have been added for all helper classes.